Fix additional tutorials (#388)
cylc/cylc-doc: src/tutorial/furthertopics/queues.rst

.. note::

>    In graph sections ``&`` is a line continuation character i.e. the

Reviewer: ```suggestion
   In graph sections, lines can be split on ``&``, i.e. the
```

Reviewer 2: (Or on `|` and `=>`)

Reviewer: Already mentioned a few lines later

   following two examples are equivalent:

   .. code-block:: cylc-graph

      foo => bar &
             baz

   .. code-block:: cylc-graph

      foo => bar & baz

   ``|`` (or), and ``=>`` act in the same way.

Validate, install and play the workflow, then open the ``cylc gui``::

   cylc validate .
   cylc install --run-name without-queues
   cylc play queues-tutorial/without-queues
   cylc gui

You will see that all the ``steak``, ``pasta``, and ``pizza`` tasks are run
at once, swiftly followed by all the ``ice_cream``, ``cheesecake``,
``sticky_toffee`` tasks as the customers order from the dessert menu.

(If you aren't very quick starting the GUI you may find that the entire
workflow has already run by the time you navigate to it.)

> 

Reviewer: Why not tell them to play from the UI instead

Author: Because the UI may not be installed https://github.com/cylc/cylc-doc/pull/388#discussion_r796746372

Reviewer: But you're talking about using it! Instead of telling them to `cylc play queues-tutorial/without-queues`, why not tell them to play the workflow using whichever UI?

Author: I've made a change which I think fixes this.

               limit = 2  # Only 2 dessert dishes at one time.
               members = DESSERT

Install and play the workflow then open up the GUI (if you closed it)::

   cylc validate .
   cylc install --run-name tutorial-with-queues
   cylc play queues-tutorial/with-queues

>    cylc gui

Reviewer 3: Not all `cylc gui` code blocks include the "if you closed it" hint. Putting `cylc gui` into every list of commands is a bit OTT and could result in users opening multiple copies of the GUI.

Suggest introducing the GUI and how to launch it, then doing something like::

    ...
    # excluding the cylc gui command

```rst
Install and plat the workflow then :ref:`inspect it in the GUI`.
```

Also the tutorials could be done with Tui instead, which is especially useful as the GUI doesn't come with Cylc by default so may not be installed.

Perhaps we could say "view the workflow in a user interface" or something a bit more snappy.

Author: I'm happy to use TUI, it's just the red warning message at the top put me off using it. IMO it's more suitable for the sort of workflows we are running.

               limit = 2  # Only 2 dessert dishes at one time.
               members = DESSERT

Install and play the workflow::

   cylc validate .
   cylc install --run-name tutorial-with-queues

>    cylc play queues-tutorial/with-queues
> 
> Look at the workflow with :ref:`tutorial.gui` or :ref:`tutorial.tui`

Reviewer: ```suggestion

Play the workflow using the ref:`GUI <tutorial.gui>`
or :ref:`TUI <tutorial.tui>`.
```
